### DISCUSSION TITLE
PDB2PQR webserver not working for RNA molecules (with residue names RA, RC, RU, RG)

### DISCUSSION THREAD

**Discussion by srdjan125** [Electrostatics/apbs#175] [Q&A]
Hello,

I have tried to protonate RNA molecule but webserver is not working and not giving me pqr files.
Few months ago I was able to do it by changing residue names URA to RU, ADE to RA, GUA --- RG and CYT --RC. However now it is not working.

I have tried several options: keeping residue names RA/RG/RU/RC , keeping them A/G/U/C or ADE/GUA/URA/CYT but none of them is working.

 I have tried to use it on my previous RNA structure which I could convert few months ago, now it is not working.

   Can someone help me what might be the issue?

  Thank you.

**Comment by sobolevnrm** [comment]
Can you please attach an example PDB file (you'll need to change the suffix of the file to `.txt` to upload)?  Thanks.

**Comment by srdjan125** [comment]
Hello, 

Sure, I am attaching two files now: RNA.txt (structure of RNA which I wanted to protonate today) and RNA_previous.txt (structure of RNA which I could protonate few months ago but today is not working).

   For now the only solution I could find is to protonate protein with PDB2PQR and RNA with H++ webserver, but I need to check if APBS will accept those PQR files generated by different programs.

________________________________

**Reply by srdjan125** [reply]
I am attaching them now
[RNA_previous.txt](https://github.com/Electrostatics/apbs/files/7245274/RNA_previous.txt)
[RNA.txt](https://github.com/Electrostatics/apbs/files/7245277/RNA.txt)

**Comment by srdjan125** [comment]
Hello,

Is there any update why online webserver is not working anymore to convert RNA PDB file into PQR? Did something changed in the meantime and residue names need to be different?

________________________________

**Comment by sobolevnrm** [comment]
The version of PDB2PQR used by the webserver has changed over the past few months.  Unfortunately, we have not had time to investigate the problem with the current version of the server.  However, it is on our "todo" list.
